CryptoLightNewsTests/CryptoLightNewsTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CryptoLightNewsTests.swift
+//  CryptoLightNewsTests
+//
+//  Created by Richard Price on 02/01/2023.
+//
+
+import XCTest
+
+@testable import CryptoLightNews
+
+
+final class CryptoLightNewsTests: XCTestCase {
+    
+    var viewController: NewsViewController!
+    // var mockAPIService: DataFetcherService!
+    
+    override func setUp() {
+        super.setUp()
+        viewController = NewsViewController()
+        //mockAPIService = DataFetcherService(networkService: Service.init())
+        //viewController.service = mockAPIService
+        viewController.loadViewIfNeeded()
+    }
+    
+    override func tearDown() {
+        viewController = nil
+        //mockAPIService = nil
+        super.setUp()
+    }
+    
+    func testItemPropertyTitleWhenSet() {
```

Comment:
Does this test belong here? 🤔  I see that you don't even use the `viewController`, which is what you are testing... Should you create another file to test just the cell?